tests/benchmarks/multicore/idle_with_pwm/testcase.yaml

```diff
@@ -0,0 +1,31 @@
+common:
+  sysbuild: true
+  tags: ci_build ci_tests_benchmarks_multicore
+  platform_allow:
+    - nrf54h20dk/nrf54h20/cpuapp
+  integration_platforms:
+    - nrf54h20dk/nrf54h20/cpuapp
+
+tests:
+  benchmarks.multicore.idle_with_pwm.nrf54h20dk_cpuapp_cpurad.no_sleep:
+    extra_args:
+      SB_CONF_FILE=sysbuild/nrf54h20dk_nrf54h20_cpurad.conf
+    harness: console
+    harness_config:
+      type: multi_line
+      ordered: true
+      regex:
+        - "Multicore idle_with_pwm test on"
+        - "Multicore idle_with_pwm test iteration 0"
+        - "Multicore idle_with_pwm test iteration 1"
+
+  benchmarks.multicore.idle_with_pwm.nrf54h20dk_cpuapp_cpurad.s2ram:
+    extra_args:
+      SB_CONF_FILE=sysbuild/nrf54h20dk_nrf54h20_cpurad.conf
+      idle_with_pwm_CONF_FILE=prj_s2ram.conf
+      remote_CONF_FILE=prj_s2ram.conf
+    harness: pytest
+    harness_config:
+      fixture: ppk_power_measure
+      pytest_root:
+        - "${CUSTOM_ROOT_TEST_DIR}/test_measure_power_consumption.py::test_measure_and_data_dump_power_consumption_simple"
```

Comment:
This test_measure_and_data_dump_power_consumption_simple checks only sleeping state (it expects no wakeups).
For evaluation, you can use test_measure_and_data_dump_placeholder which only measures current for 10 sec (and dumps data).